src/apps/users/api/password.py

```diff
@@ -72,7 +72,8 @@ async def password_recovery(
     async with atomic(session):
         try:
             content_source = await get_mindlogger_content_source(request)
-            await PasswordRecoveryService(session).send_password_recovery(schema, content_source)
+            content_language = request.headers.get("content-language", "en")
```

Comment:
Good way to handle this 💯 

I'm seeing an error when attempting to call password reset in French. This error doesn't occur for English or Greek.

```File "/Users/carloschacon/Developer/mindlogger-backend-refactor/src/apps/users/api/password.py", line 72, in password_recovery
    async with atomic(session):
  File "/Users/carloschacon/Developer/mindlogger-backend-refactor/src/apps/users/api/password.py", line 76, in password_recovery
    await PasswordRecoveryService(session).send_password_recovery(schema, content_source, content_language)
  File "/Users/carloschacon/Developer/mindlogger-backend-refactor/src/apps/users/services/core.py", line 96, in send_password_recovery
    body=service.get_localized_html_template(
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/carloschacon/Developer/mindlogger-backend-refactor/src/apps/mailing/services.py", line 71, in get_localized_html_template
    return self.get_localized_html_template(template_name, "en", **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: MailingService.get_localized_html_template() got multiple values for argument 'language'
```